homeassistant/components/matter/light.py

```diff
@@ -78,6 +79,23 @@ def supports_brightness(self) -> bool:
             return False
         return ColorMode.BRIGHTNESS in self._attr_supported_color_modes
 
+    def supports_color_feature(self, feature: ColorControlCapabilities) -> bool:
+        """Return if the device supports the given color feature."""
+        featuremap = self.get_matter_attribute_value(
```

Comment:
maybe just do the initial lookup for the featuremap here and set it into a attribute ?
something like if _matter_color_feature_map is None -> do lookup of the value ?